pyproject.toml

```diff
@@ -20,6 +20,7 @@ mkdocs = {version="^1.3.0", optional = true}
 mkdocs-material = {version="^8.2.9", optional = true}
 mkdocstrings = {version="^0.19.0", optional = true, extras=["python"]}
 mkdocs-jupyter = {version="^0.20.1", optional = true}
+toml = "^0.10.2"
 
```

Comment:
@Cr0n1c dependencies are rigid due to the data tools extra.
It's required to hit our priority to enable usage of godel in google colab and other workspaces (they use py3.7 bleh).
I'll spend some time to figure out if i can ease some dependencies